source/documentation/security.md

```diff
@@ -82,4 +82,6 @@ Your service manager may also be asked to supply extra evidence on your internal
 
 GOV.UK Pay follows [government HTTPS security guidelines](https://www.gov.uk/service-manual/domain-names/https.html). The Hypertext Transfer Protocol Secure (HTTPS), which involves the Transport Layer Security  (TLS) protocol is used by the platform to authenticate servers / clients and to provide secure connections.
 
-Your government service will only be able to integrate with the GOV.UK Pay if it also uses HTTPS.
+To take a payment with a live GOV.UK Pay account, you must use HTTPs. 
+
+To take a payment with a sandbox account, [you can use HTTP](/testing_govuk_pay/#testing-gov-uk-pay).
```

Comment:
I think this could benefit some context to say it really relates to the return URL. If that’s too specific, perhaps it could say that a service that integrates with Pay has to use HTTPS but there’s an exception for sandbox accounts to make development easier.

Reply:
I'm fine with either but not quite sure how to phrase it if we go with the first. Do you think the first would be more useful to the user?

Reply:
In terms of HTTPS, there’s two things really.

Firstly, all direct communication between the service and Pay must be HTTPS. This is implied by the API URLs and usually not onerous to do, so it doesn’t require much calling out.

The second aspect is that the return URLs must be HTTPS (except for sandbox). This is something we know can be a pain and the requirement has surprised people in the past (which is why we added the sandbox exemption).

If we just say you have to use HTTPS, I think a lot of people will only think about the communication between the service and Pay and not realise we mean the return URLs as well (except the sandbox exemption).